modules/nf-core/modules/samtools/merge/main.nf

```diff
@@ -22,8 +22,8 @@ process SAMTOOLS_MERGE {
     tuple val(meta), path(bams)
 
     output:
-    tuple val(meta), path("${prefix}.bam"), emit: merged_bam
-    path  "*.version.txt" ,                 emit: version
+    tuple val(meta), path("${prefix}.bam"), emit: bam
```

Comment:
also back to bams here...I think it is jsut a naming thing. I guess, that is how they come from nf-core/modules

Reply:
actually we have the samtools/merge_cram module in local, we probably don't need both, I'll check this out.
Thanks for the review

Reply:
ok, so we need both for now, but it'll be nice to have only one module, I'll look this up in a separate PR